app/code/Magento/CurrencySymbol/Test/Mftf/ActionGroup/StorefrontCurrencyRatesActionGroup.xml

```diff
@@ -17,13 +17,4 @@
         <click selector="{{StorefrontSwitchCurrencyRatesSection.currency(currency)}}" stepKey="chooseCurrency"/>
         <see selector="{{StorefrontSwitchCurrencyRatesSection.selectedCurrency}}" userInput="{{currency}}" stepKey="seeSelectedCurrency"/>
     </actionGroup>
-    <actionGroup name="StorefrontSwitchCurrency">
```

Comment:
Nice catch here :)
It looks like the result of incorrect merge conflict resolving.
I guess we can also remove `currencyTrigger` element from `StorefrontSwitchCurrencyRatesSection` section, to avoid duplicates and misunderstandings in the future. @lbajsarowicz what do you think?

Reply:
I removed that to avoid confusion.
I'll review all the sections soon during other Contribution Days :-)